card.py

```diff
@@ -1,29 +1,81 @@
 # -*- coding: utf-8 -*-
 """
+Class to create card objects
 Created on Fri Oct 22 12:46:28 2021
-"""
-
-from enum import Enum
 
-class Color(Enum):
-    RED = "red"
-    GREEN = "green"
-    BLUE = "blue"
-    YELLOW = "yellow"
-    WILD = "wild"
+@author: Fabio Tran
+"""
 
+# Class to create card objects
 class Card:
     
-    def __init__(self, clr, val):
-        color = clr
-        value = val
-        print("{0} {1}".format(color.value , value))
+    # Constructor
+    def __init__(self, color, number, type):
+        self.color = color
+        self.number = number
+        self.type = type
+        self.wild_color = None
         
-    def match(c1, c2):
-        if(c1.color == c2.color or c1.value == c2.value):
+    # Sets color of wild card once it is played 
+    def set_wild_color(self, color):
+        self.wild_color = color
+
+     # Used to compare color or number between two cards 
+    def match_card(self, other_card):
+        # checks if other card is a wild card
+        if(other_card.color == "Any"):
```

Comment:
Some more detailed comments about what self and other_card are would help make this clearer. Basically explaining that self is supposed to be the last played card and other is the card being played.